src/pages/Encounters/EncounterShow.tsx

```diff
@@ -300,6 +300,7 @@ export const EncounterShow = (props: Props) => {
               <div className="font-base flex flex-col text-xs leading-relaxed text-secondary-700">
                 <div className="flex items-center">
                   <span className="text-secondary-900">Created: </span>&nbsp;
```

Comment:
let's not have this since backend is not giving the data. period.start is already shown at the top.

remove this